### DISCUSSION TITLE
Upcoming Webinar- Improving MOOSE workflows through Coreform Cubit

### DISCUSSION THREAD

**Discussion by jf503** [idaholab/moose#20708] [News]
On **April 8th** (9 am Mountian) Coreform will introduce two MOOSE-related projects they have been working on for the past few years: adding isogeometric analysis (IGA) to MOOSE, and a MOOSE editor integrated with Coreform Cubit. The MOOSE editor integrated with Coreform Cubit has been designed to accelerate the creation and debugging of MOOSE input files.  Both of these features are in a beta state and will be released later this year; Coreform is seeking early adopters who are interested in testing the functionality and providing feedback as we polish.

[Register here ](https://us02web.zoom.us/webinar/register/WN_Ls2D_0MJTv6-l_MrTGl9iQ)

![MOOSE](https://user-images.githubusercontent.com/15050616/161612716-fc09dcad-39aa-499f-a37c-9bcb9a4d5768.png)

**Comment by Traiwit** [comment]
awesome! but this will be at 1 am Sydney, Australia time, hope you guys have a recording of the session afterwards :)

**Reply by GregVernon** [reply]
Yes, we record all our webinars and post them to our [YouTube channel](https://www.youtube.com/c/CoreformLLC) shortly after - usually within 48 hours.

**Reply by Traiwit** [reply]
awesome, thank you!

**Comment by GregVernon** [comment]
@Traiwit -- The webinar has been uploaded: https://www.youtube.com/watch?v=UmS-pAuBnEU
